meshkernel/meshkernel.py

```diff
@@ -282,6 +282,39 @@ def mesh2d_delete(
             c_int(invert_deletion),
         )
 
+    def mesh2d_get_inner_boundary_polygons(self) -> GeometryList:
```

Comment:
There is much similarity with `mesh2d_get_mesh_boundaries_as_polygons()`. I think it would be useful to use the same naming convention, so that would be `mesh2d_get_mesh_inner_boundaries_as_polygons()`. I am not necessarily a big fan of these long names, but I am a big fan of consistency.